fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/DeriveStatsTask.java

```diff
@@ -61,6 +63,27 @@ public void execute() {
                 (expressionContext.getStatistics().getOutputRowCount() < currentStatistics.getOutputRowCount())) {
             groupExpression.getGroup().setStatistics(expressionContext.getStatistics());
         }
+        if (currentStatistics != null && !currentStatistics.equals(expressionContext.getStatistics())) {
+            if (groupExpression.getOp() instanceof LogicalOlapScanOperator) {
+                LogicalOlapScanOperator scan = groupExpression.getOp().cast();
+                if (scan.getTable().isMaterializedView()) {
+                    PhysicalOlapScanOperator physicalOlapScan = new PhysicalOlapScanOperator(
+                            scan.getTable(),
+                            scan.getColRefToColumnMetaMap(),
+                            scan.getDistributionSpec(),
+                            scan.getLimit(),
+                            scan.getPredicate(),
+                            scan.getSelectedIndexId(),
+                            scan.getSelectedPartitionId(),
+                            scan.getSelectedTabletId(),
+                            scan.getPrunedPartitionPredicates(),
+                            scan.getProjection());
```

Comment:
save table id is enough?

Reply:
no, one mv can be used in different groups, so should use GroupExpression as key